### ISSUE TITLE
Github repository is not updated anymore

### ISSUE THREAD

**Issue by rozie** [firehol/blocklist-ipsets#270]
Probably due to change of Github SSH keys, lists in this repository are not updated anymore.

**Comment by alyandon** [comment]
Yeah, https://github.com/firehol/blocklist-ipsets/issues/263 and no response from maintainers.

Seems like this project is on autopilot and slowly dying on the vine.

**Comment by e-lisa** [comment]
Use http://iplists.firehol.org/ to download the lists. Good luck & have fun!

**Comment by rozie** [comment]
That's what I do, but downloading from GH was better. Or maybe I'm just missing one file archive containing all lists in separate files?

**Comment by e-lisa** [comment]
idk ask the maintainer of the project

**Comment by borestad** [comment]
I've added a firehol mirror here:
https://github.com/borestad/firehol-mirror
